extensions/google-workspace/src/components/FileListItem.tsx

```diff
@@ -75,6 +75,20 @@ export default function FileListItem({ file, email }: FileListItemProps) {
               title="Copy File URL"
               shortcut={{ modifiers: ["cmd", "shift"], key: "," }}
             />
+
+            <Action.CopyToClipboard
+              icon={Icon.Clipboard}
+              content={`[${file.name}](${file.webViewLink})`}
+              title="Copy Markdown Link"
+              shortcut={{ modifiers: ["cmd", "shift"], key: "." }}
+            />
+
+            <Action.CopyToClipboard
+              icon={Icon.Clipboard}
+              content={`<a href="${file.webViewLink}" title="${file.name}">${file.name}</a>`}
+              title="Copy HTML Link"
+              shortcut={{ modifiers: ["cmd", "shift"], key: "/" }}
+            />
```

Comment:
You need to pass an object for HTML links otherwise this won't be properly copied as an HTML link:

```suggestion
            <Action.CopyToClipboard
              content={{
                html: `<a href="${file.webViewLink}" title="${file.name}">${file.name}</a>`,
                text: file.name
              }}
              title="Copy HTML Link"
              shortcut={{ modifiers: ["cmd", "shift"], key: "/" }}
            />
```